src/applications/vaos/services/appointment/transformers.js

```diff
@@ -438,12 +441,13 @@ function createPatientResourceFromRequest(req) {
  * Builds contained array and populates with video conference info and facility info if available
  *
  * @param {Object} appt  VAR appointment object
- * @returns {Array} Array of contained objects of FHIR appointment containing video conference info
+ * @returns {Array|Object} Returns array of contained objects of FHIR appointment containing video conference info or
+ * returns an object containing patient information.
  */
 function setContained(appt) {
   switch (getAppointmentType(appt)) {
     case APPOINTMENT_TYPES.request: {
-      return [createPatientResourceFromRequest(appt)];
+      return createPatientResourceFromRequest(appt);
     }
     case APPOINTMENT_TYPES.ccRequest: {
       const contained = [createPatientResourceFromRequest(appt)];
```

Comment:
It might be easier to convert this one to the single patient object, too, since cc requests and va requests are very similar.

Reply:
Done